packages/x-charts/src/ChartsTooltip/ChartsItemTooltipContent.tsx

```diff
@@ -31,15 +31,19 @@ export type ChartsItemContentProps<T extends ChartSeriesType = ChartSeriesType>
    */
   getColor: (dataIndex: number) => string;
   sx?: SxProps<Theme>;
-};
+}
 
-function ChartsItemTooltipContent<T extends ChartSeriesType>(props: {
+export interface ChartsItemTooltipContentProps<T extends ChartSeriesType> {
```

Comment:
It's correctly types, but not correctly proptyped

To have this working we would need to override proptypes in the pro packages

Reply:
🤔 true, are we able to properly override it though?

Reply:
It would be possible do something like 

```jsx
import { ChartsTooltip as ChartsTooltipCommunity, ChartsTooltpiProps } from '@mui/x-charts';

export function ChartsTooltip(props: ChartsTooltipProps<SeriesType>) { return <ChartsTooltipCommunity {...props} />}

```
This would allow to get the correct proptypes.

Another solution would be to generate the proptypes with tht TS typing coming from the pro component

A last solution( the one I picked) is to remove the proptypes from the problematic component which btw should pribably be internals.

I chose the last one because proptypes will probably soon disappear, so does not worth spending time on fixing it